src/application/ventilation.c

```diff
@@ -0,0 +1,80 @@
+/*
+ * ventilation.c
+ *
+ */
+
+#include <stdbool.h>
+#include <platform/systemInfo.h>
+#include <platform/pressureFlowSensor.h>
+#include <platform/valve.h>
+#include <application/ventilation.h>
+#include <application/dss.h>
+
+#define INSPIRATORY_FLOW_TIME 	1
+#define INSPIRATORY_PAUSE	2
+#define EXPIRATORY_FLOW_TIME 	3
+#define EXPIRATORY_PAUSE	4
+
+/*
+ * the time the current breath cycle was supposed to start
+ */
+static int start_current_breath_cycle = 0;
+
+/*
+ * the time the current breath cycle was supposed to start
+ */
+static int breath_cycle_duration = 125; //to do replace this calue by the real one
+
+/*
+ * Phase of the ventilation
+ * This should always be one of
+ *  INSPIRATORY_FLOW_TIME | INSPIRATORY_PAUSE
+ *  | EXPIRATORY_FLOW_TIME | EXPIRATORY_PAUSE
+ */
+
+int get_circuit_pressure() { return 0; }
+
+static int ventilation_phase = EXPIRATORY_PAUSE;
+
+void ventilation(){

```

Comment:
I think you should preface this with the name `task_ventilation()`. We agreed that the task functions should be prefaced with `task_` so they are obvious in the rest of the code.